### ISSUE TITLE
None/none/null default value in plasma configuration schema

### ISSUE THREAD

**Issue by ftsamis** [tardis-sn/tardis#633]
Something that had occurred when I was converting `tardis_config_definition.yml` to JSON Schema was the default values of `None` which sometimes were mistakenly expressed as `none`.

When the word None didn't start with a capital `N`, the (old) validator translated it as a string. The two cases were this happened are `plasma.helium_treatment` and `plasma.heating_rate_data_file`

For example, in the `plasma.heating_rate_data_file` case its default value was set to `none` but [the code](https://github.com/tardis-sn/tardis/commit/7fb1a678c724b02bead8d11bf9fb7923705ec4a2#diff-1418433c6e9468b4261cee12840d02daR113) is comparing to the python object `None`, making the code have no effect.

This bug has survived after the conversion to JSON Schema, since only what the old validator was translating as python `None` was converted to jsonschema `null` (which is the way to say python `None` in jsonschema). This was left out on purpose because it was out of scope for the particular task, and changes in the code needed to be done to solve it.

Finally, we should discuss dropping None/null as a default value altogether, since the only thing it offers is not having to check in the code if the property was provided by the configuration file, i.e. it only saves us from doing:

``` python
if 'helium_treatment' in config.plasma:
    helium_treatment = config.plasma.helium_treatment
else:
    helium_treatment = None
```

 but there are nice workarounds to this, such as 

``` python
helium_treatment = config.plasma.get('helium_treatment', None)
```

**Comment by unoebauer** [comment]
Status?

**Comment by ftsamis** [comment]
This is still an issue. Let's please fix it after #652 is merged or as part of it.

**Comment by nitinkgp23** [comment]
@ftsamis I am a newcomer here, and I would like to work on this issue. Can you help me in getting forward with it?

**Comment by ftsamis** [comment]
Feel free to ask any specific questions.

**Comment by unoebauer** [comment]
Closing due to inactivity - reopen if necessary.
